github_activity/github_activity.py

```diff
@@ -206,13 +207,16 @@ def get_activity(
     # Query for both opened and closed issues/PRs in this window
     print(f"Running search query:\n{search_query}\n\n", file=sys.stderr)
     query_data = []
+    all_bot_users = set()
     for activity_type in ["created", "closed"]:
         ii_search_query = (
             search_query + f" {activity_type}:{since_dt_str}..{until_dt_str}"
         )
         qu = GitHubGraphQlQuery(ii_search_query, auth=auth)
         qu.request()
         query_data.append(qu.data)
+        # Collect bot users from each query
+        all_bot_users.update(qu.bot_users)
 
     query_data = (
         pd.concat(query_data).drop_duplicates(subset=["id"]).reset_index(drop=True)
```

Comment:
this I think was clobberying bot_users so we are now explicitly grabbing it in a set after each query